generators/entity-server/templates/src/main/java/package/domain/imports.ejs

```diff
@@ -35,7 +35,7 @@ if (!importApiModelProperty) {
 }
 
 for (idx in relationships) {
-    if (relationships[idx].relationshipType === 'one-to-many'
+    if (relationships[idx].relationshipType === 'one-to-many' && relationships[idx].ownerSide === false
```

Comment:
Could you please factor the whole condition ? 
```
relationships[idx].ownerSide === false && ['one-to-many', 'one-to-one', 'many-to-many'].includes(relationships[idx].relationshipType)
```

Reply:
Yes, I'll do that